boards/common/silabs/Makefile.dep

```diff
@@ -1,3 +1,8 @@
 ifneq (,$(filter silabs_pic,$(USEMODULE)))
   USEMODULE += periph_i2c
 endif
+ifneq (,$(filter gnrc_netdev_default netdev_default,$(USEMODULE)))
```

Comment:
I don't have a strong opinion (because of `EFM32_RADIO`) about whether this should be here, or in the CPU (e.g. [here](https://github.com/kaibeckmann/RIOT/blob/8e710673853c93a712bd37d618ec032f82c9fb50/cpu/efm32/Makefile.features)). @kYc0o?

Reply:
Also only `netdev_default` should be required.